### DISCUSSION TITLE
@UseQt disables

### DISCUSSION THREAD

**Discussion by GhiathMonnet** [thelfer/tfel#578] [General]
Dear All, while moving to mfront version distributed through salome-meca (mfront version 3.4.3 and 3.4.0), I got the following message 

Error treating the  keyword 'UseQt'. The keyword '@UseQt' has been disabled   

Do you know what that means ?
Can I do something about that ?
Many thanks.
Cheers

**Comment by thelfer** [comment]
Hi @GhiathMonnet,
TFEL versions 3.4 are now 4 years old. Support for quantities was experimental. In particular, those versions did not have proper support for quantities in all DSLs, in particular the isotropic ones.
There is not much you can do about this, except commenting `@UseQt`...

**Reply by GhiathMonnet** [reply]
OK I understand. Thank you Thomas 👍
